tests/components/fieldsets/structure/FieldSetAdresseAdministrative.test.tsx

```diff
@@ -0,0 +1,310 @@
+import { render, screen, waitFor } from "@testing-library/react";
+import userEvent from "@testing-library/user-event";
+import { describe, expect, it, vi } from "vitest";
+
+import { FieldSetAdresseAdministrative } from "@/app/components/forms/fieldsets/structure/FieldSetAdresseAdministrative";
+import { Repartition } from "@/types/adresse.type";
+import { FormKind } from "@/types/global";
+
+import { FormTestWrapper } from "../../../test-utils/form-test-wrapper";
+
+vi.mock("@/app/hooks/useAddressSuggestion", () => ({
+  useAddressSuggestion: () => {
+    return async () => [];
+  },
+}));
+
+describe("FieldSetAdresseAdministrative", () => {
+  describe("Rendering with FormKind.FINALISATION", () => {
```

Comment:
```suggestion
  describe("Rendering finalisation form", () => {
```
Pour essayer de retirer les notions techniques d'enum de l'intitulé du test qui peut être un peu plus "abstrait"